public/app/features/alerting/unified/mocks/server/handlers/mimirRuler.ts

```diff
@@ -0,0 +1,74 @@
+import { delay, http, HttpResponse } from 'msw';
+
+import { RulerRuleGroupDTO } from '../../../../../../types/unified-alerting-dto';
+import { namespaces } from '../../mimirRulerApi';
+import { HandlerOptions } from '../configure';
+import { MIMIR_DATASOURCE_UID } from '../constnts';
+
+export const updateRulerRuleNamespaceHandler = (options?: HandlerOptions) => {
+  return http.post<{ namespaceName: string }>(
+    `/api/ruler/${MIMIR_DATASOURCE_UID}/api/v1/rules/:namespaceName`,
```

Comment:
As we explicitly name the grafana handlers as `/api/ruler/grafana/...`, and those handlers are used before the mimir ones in our mock server, it would be better if this URL also included the datasource UID as a parameter, and then we can instead more easily do dynamic responses based on the datasource that we asked about

`/api/ruler/:datasourceUid/api/v1/rules/:namespace`